django/cantusdb_project/main_app/management/commands/sync_source_creation_date.py

```diff
@@ -0,0 +1,87 @@
+from django.core.management.base import BaseCommand
+from main_app.models import (
+    Source,
+)
+import requests, json
+import datetime
+from django.utils import timezone
+
+
+def get_source_list():
+    source_list = []
```

Comment:
The body of this function can be simplified:

```py
with open("source_list.txt", "r") as myfile:
    source_list = myfile.read().splitlines()

return source_list
```

Using a context manager (`with ...`) on a file object will automatically close the file when it goes out of scope.